### PR TITLE
Sonata ckeditor

### PR DESCRIPTION
**Pull request by christianJCK**
Hey!
I finished the WYSIWYG.
I wanted to know if you can check it.

I still have an issue with the date format but i will "issue" it and look at it later as it is related to the ckeditor.

thank you @Gianni-n @ngdo-pro @Nagx

### CODE REVIEW

File: src/BookEditorBundle/Admin/BookAdmin.php [hunk 1]
```diff
@@ -20,9 +20,9 @@ protected function configureFormFields(FormMapper $formMapper)
                 'label' => 'Auteur',
                 'required' => true
             ))
-            ->add('description', 'text', array(
-                'label' => 'Description',
-                'required' => true
+            ->add('description', 'ckeditor', array(
+                'config_name' => 'my_config',
+                'config'      => array('uiColor' => '#ffffff', 'language' => 'fr')
```
Comment (ngdo-pro) [inline src/BookEditorBundle/Admin/BookAdmin.php]: can you add the label with 'Description' ? to still have the uppercase

File: src/BookEditorBundle/Admin/BookAdmin.php [hunk 2]
```diff
@@ -40,7 +40,9 @@ protected function configureFormFields(FormMapper $formMapper)
                 'label' => 'Image de l\'article de presse',
                 'required' => false
             ))
-            ->add('releaseDate', 'date', array(
+            ->add('releaseDate', 'date',array(
+                'years' => range(2000, 2020),
+                'format' => 'ddMMMMyyy',
```
Comment (ngdo-pro) [inline src/BookEditorBundle/Admin/BookAdmin.php]: just a question about that line : why is there 4 "M" ? i mean, months only have 2 numbers

File: src/BookEditorBundle/Admin/BookAdmin.php [hunk 3]
```diff
@@ -60,7 +62,14 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
...
             ->add('title')
+            ->add('author')
             ->add('description')
+            ->add('facebookLinkUrl')
+            ->add('imageUrl')
+            ->add('pressTitle')
+            ->add('pressImageUrl')
+            ->add('releaseDate')
+            ->add('purchaseOrderImageUrl')
         ;
```
Comment (ngdo-pro) [inline src/BookEditorBundle/Admin/BookAdmin.php]: you added too many filters imo, i think you should keep it low, only 3-4 filters, we need to choose them right

File: src/BookEditorBundle/Admin/BookAdmin.php [hunk 4]
```diff
@@ -69,7 +78,14 @@ protected function configureListFields(ListMapper $listMapper)
     {
         $listMapper
```
Comment (ngdo-pro) [inline src/BookEditorBundle/Admin/BookAdmin.php]: same here, those are the fields that will appear in the list, i don't think we need all the fields here

**Comment by ngdo-pro** [comment]
:+1:

**Comment by Nagx** [comment]
:+1:

**Comment by Gianni-n** [comment]
nice :+1:

**Pull request merged**